src/terms

```diff
@@ -0,0 +1,89 @@
+We reserve the right to terminate the Services at our sole discretion or to transfer operation of the Services to a third party or smart contract. Prior to termination or transfer, we will provide 90 days’ notice to users via email to allow users enough time to make arrangements if any (e.g. remote pinning to their own IPFS node, remote pinning to another service, or preparing to manage the renewal of their Filecoin deals on their own).
+
+All data uploaded to web3.storage is available to anyone who requests it using the correct CID. **Users should not store any private or sensitive information in an unencrypted form using web3.storage.** Deleting files from web3.storage via the site's web console or API will remove them from the file listing for a user's account, but you may be charged for at least 30 days of storage for those files. Additionally, uploaded files typically remain on our infrastructure for at minimum thirty (30) days, and nodes on the IPFS network may retain copies of the data indefinitely. **Users should not use web3.storage to store data that may need to be permanently deleted in the future.**
+
+#### **Paid Accounts**
+
+<u>Billing.  </u>
+ In order to begin storing files via the Services, you will need to provide us with one or more Payment Methods. "Payment Method" means a current, valid method of payment, which may involve a third party or payment processing vendor. You can increase your storage limit and add paid features to your account by signing up for a paid subscription plan (“Paid Plan”). 
+
+You agree that you’ll pay the amounts set forth in https://web3.storage/account/payment/ for your chosen tier (“Subscription Charge”) plus any charges for additional usage beyond the storage limit of your selected tier during the previous month (“Overages”) (collectively, “Subscription Charge” and “Overages” are referred to as (the “Fees”) for each billing cycle). 
+
+When determining tier storage limits and Overages, we track usage for payment using the volume of data associated with shard content identifiers/CIDs (CIDs referencing the actual data sent to web3.storage and stored) – usage is not tracked based on content CIDs (CIDs used to reference and access uploads in a format useful to users, like specific files and directories). See more detail on this point here(ADD LINK).
+
+You authorize us to charge any Payment Method associated with your account in case your primary Payment Method is declined or no longer available to us for payment of Fees you owe us. You remain responsible for any uncollected Fees. If a payment is not successfully settled, due to expiration, insufficient funds, or otherwise, and you do not cancel your account, we may suspend your access to the Paid Plan until we have successfully charged a valid Payment Method. For some Payment Methods, the issuer may charge you additional fees, such as foreign transaction fees or other fees relating to the processing of your Payment Method.
+
+<u>Cancellation.  </u>
```

Comment:
these all look like h3 items
```suggestion
### Cancellation.
```

Reply:
Again, copying format from received document. It was underlined.